runelite-client/src/main/java/net/runelite/client/plugins/info/InfoPanel.java

```diff
@@ -52,6 +63,14 @@
 {
 	private static final String RUNELITE_LOGIN = "https://runelite_login/";
 
+	private final JLabel loggedLabel = new JLabel();
+	private final JRichTextPane emailLabel = new JRichTextPane();
+
+	private static ImageIcon ARROW_RIGHT_ICON;
+	private static ImageIcon GITHUB_ICON;
+	private static ImageIcon DISCORD_ICON;
+	private static ImageIcon PATREON_ICON;
```

Comment:
Mark these 4 fields final too and move all constants together at top of the class